appbuilder/providers/livesync-provider.ts

```diff
@@ -0,0 +1,17 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+import Future = require("fibers/future");
+import { AppBuilderLiveSyncProviderBase } from "./appbuilder-livesync-provider-base";
+
+export class LiveSyncProvider extends AppBuilderLiveSyncProviderBase {
+	constructor($androidLiveSyncServiceLocator: {factory: Function},
+		$iosLiveSyncServiceLocator: {factory: Function}) {
+			super($androidLiveSyncServiceLocator, $iosLiveSyncServiceLocator);
+		}
+
+	public buildForDevice(device: Mobile.IDevice): IFuture<string> {
+		return Future.fromResult(null);
```

Comment:
It seems that this https://github.com/telerik/mobile-cli-lib/pull/617/files#diff-1974b437396edadf4434400ddfadc9a1R188 will fail if the result from this function is `null`.